src/bp/ui-studio/src/web/views/Config/index.tsx

```diff
@@ -0,0 +1,507 @@
+                    inputProps={{
+                      id: 'cover-picture-url',
+                      name: 'coverPictureUrl',
+                      accept: 'image/*',
+                      onChange: this.handleImageFileChanged
+                    }}
+                  />
+                  {this.state.coverPictureUrl !== this.initialFormState.coverPictureUrl && (
+                    <p className={style.configUploadSuccess}>
+                      The cover picture has been uploaded successfully. You need to save the form in order for the
+                      changes to take effect.
+                    </p>
+                  )}
+                  {this.state.coverPictureUrl && (
+                    <img className={style.coverPreview} alt="cover" src={this.state.coverPictureUrl} />
+                  )}
+                </FormGroup>
+              </div>
+            )}
+            <FormGroup>
+              <Button
+                text="Save changes"
+                intent="primary"
+                icon="floppy-disk"
+                disabled={this.state.isSaving}
+                onClick={this.saveChanges}
+              />
+            </FormGroup>
+          </form>
+        </div>
+      </Container>
+    )
+  }
+
+  renderLanguages() {
+    const languages = this.state.languages.map(l => (
+      <option key={l.value} value={l.value}>
+        {l.label}
+      </option>
+    ))
+
+    if (this.state.licensing && this.state.licensing.isPro) {
+      return (
+        <div>
+          <FormGroup label="Default language" labelFor="selected-default-lang">
+            <Select
+              id="selected-default-lang"
+              name="selectedDefaultLang"
+              options={this.state.languages}
+              value={this.state.selectedDefaultLang}
+              onChange={this.handleDefaultLangChanged}
+            />
+          </FormGroup>
+          <FormGroup label="Supported languages" labelFor="selected-languages">
+            <Select
+              id="selected-languages"
+              name="selectedLanguages"
+              options={this.state.languages}
+              value={this.state.selectedLanguages}
+              onChange={this.handleLanguagesChanged}
+              isMulti
+            />
+          </FormGroup>
+        </div>
+      )
+    } else {
+      return (
+        <FormGroup label="Language" labelFor="selected-default-lang">
+          <select
+            id="selected-default-lang"
+            name="selectedDefaultLang"
+            value={this.state.selectedDefaultLang}
+            onChange={this.handleCommunityLanguageChanged}
+          >
+            {languages}
+          </select>
+        </FormGroup>
+      )
+    }
+  }
+}
+
+const mapStateToProps = state => ({ bot: state.bot })
+
+const mapDispatchToProps = {
+  fetchBotInformation
+}
+
+export default connect(mapStateToProps, mapDispatchToProps)(ConfigView)
+
+type Props = {
+  fetchBotInformation: Function
+  bot: any
+}
```

Comment:
Please use `interface Props {}` instead when possible, to stay consistent with the rest of the codebase. We use type mostly to extend a definition

Also, shouldn't bot be typed as BotConfig ?